ui/src/Credentials.ts

```diff
@@ -27,11 +27,11 @@ function isCredentials(credentials: any): credentials is Credentials {
 
 const CREDENTIALS_STORAGE_KEY = 'credentials';
 
-export function storeCredentials(credentials?: Credentials): void {
+export function storeCredentials(credentials?: PartyToken): void {
```

Comment:
since we now have `Credentials` and `PartyToken` should we rename the arg `partyToken?: PartyToken `?

Reply:
Actually I moved this back to `Credentials`. I ought to be more careful because `PartyToken` is Hub-specific, so `Credentials` is actually more general (i.e. for local dev) 

i revised the PR to try to make sure `PartyToken` stuff is only contained within parts of the code that are only supposed to run on Daml Hub